ssl/src/main/java/io/crate/protocols/postgres/ssl/SslReqHandlerUtils.java

```diff
@@ -25,7 +25,7 @@
 import io.netty.channel.Channel;
 import org.elasticsearch.common.settings.Settings;
 
-public class SslHandlerUtils {
+public class SslReqHandlerUtils {
 
     public static SslReqHandler getDefault() {
```

Comment:
Is this now only used in tests? If so I think we should remove it here and move it into the test or just inline the constructor call